lib/reda/configs/configManager.py

```diff
@@ -113,6 +113,35 @@ def load_configs(self, filename):
         configs = np.loadtxt(filename)
         self.add_to_configs(configs)
 
+    def load_injections_from_mcf(filename):
+        """Load injections from a mcf-file used by the sEIT-systems from FZJ;
+        Injections in the file must be formatted like the following:
+        SE 001 002
+        SE 002 001
+        .
+        .
+        .
+
+        Parameters
+        ----------
+        filename: string
+            absolute or relative path to a mcf-file
+        
+        Returns
+        -------
+        Nx2 numpy array with current injections
+        """
+
+        injections = []
+        #load all injections
+        with open(filename, encoding="latin-1") as mcf:
+            for line in mcf:
+                if line[:2] == "SE":
+                    injections.append((line[3:6],line[7:10]))
+        #remove every second line to remove double entries
```

Comment:
Could you add a check for the swapped injections? Sometimes we import data that does not follow this practice (by design or by mistake).

Something like
`if injections[1::2] ==injections[0::2]...`